lib/cli.js

```diff
@@ -24,21 +25,46 @@ module.exports = function broccoliCLI(args) {
     .option('--port <port>', 'the port to bind to [4200]', 4200)
     .option('--host <host>', 'the host to bind to [localhost]', 'localhost')
     .option('--brocfile-path <path>', 'the path to brocfile')
+    .option('--output-path <path>', 'the path to target output folder')
     .action(options => {
       actionPerformed = true;
 
+      const outputDir = options.outputPath;
       const builder = getBuilder({ brocfilePath: options.brocfilePath });
+      const watcher = new Watcher(builder);
+
+      if (outputDir) {
+        if (fs.existsSync(outputDir)) {
```

Comment:
Can we make a shared function in module scope for this assertion? It seems to be the same between here and the `build` command...

Reply:
done